test/regressions/.mocharc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   recursive: true,
+  slow: 500,
```

Comment:
The default is 75ms.

<img width="577" alt="Screenshot 2021-02-21 at 17 35 56" src="https://user-images.githubusercontent.com/3165635/108631677-4ef60200-746b-11eb-84ec-36e3acaef812.png">

Considering screenshots add, it seems 200ms. I thin that below 250ms is fast